glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt

```diff
@@ -488,17 +488,47 @@ open class GleanInternalAPI internal constructor () {
     }
 
     /**
-     * Should be called from all users of the Glean testing API.
+     * TEST ONLY FUNCTION.
+     * This is called by the GleanTestRule, to enable test mode.
      *
      * This makes all asynchronous work synchronous so we can test the results of the
      * API synchronously.
      */
-    @VisibleForTesting(otherwise = VisibleForTesting.NONE)
-    fun enableTestingMode() {
+    internal fun enableTestingMode() {
         @Suppress("EXPERIMENTAL_API_USAGE")
         Dispatchers.API.setTestingMode(enabled = true)
     }
 
+    /**
+     * TEST ONLY FUNCTION.
+     * Resets the Glean state and trigger init again.
+     *
+     * @param context the application context to init Glean with
+     * @param config the [Configuration] to init Glean with
+     * @param clearStores if true, clear the contents of all stores
+     */
+    internal fun resetGlean(
+        context: Context,
+        config: Configuration,
+        clearStores: Boolean
+    ) {
+        Glean.enableTestingMode()
+
+        /*if (clearStores) {
+            // Clear all the stored data.
+            val storageManager = StorageEngineManager(applicationContext = context)
+            storageManager.clearAllStores()
+            // The experiments storage engine needs to be cleared manually as it's not listed
+            // in the `StorageEngineManager`.
+            ExperimentsStorageEngine.clearAllStores()
+        }*/
+
+        // Init Glean.
+        // Glean.setUploadEnabled(true)
```

Comment:
Should this be uncommented?

Reply:
You're right. This was moved, so it was like that before, but I think it makes sense to flip this on to behave exactly as glean-ac.